src/network-interface.js

```diff
@@ -4,7 +4,10 @@ import {extractRequestFiles} from './helpers'
 export class HTTPUploadNetworkInterface extends HTTPFetchNetworkInterface {
   fetchFromRemoteEndpoint ({request, options}) {
     // Skip upload proccess if SSR
-    if (typeof window !== 'undefined') {
+    if (
+      (typeof window !== 'undefined') ||
+      (typeof navigator != 'undefined' && navigator.product == 'ReactNative')
```

Comment:
`!=` is probably a linting error.

Perhaps we should just create one set of checks at the top of the module to define an agent const, and use it throughout the code to make it more DRY and readable.